src/metrics/include/metrics/MetricsReporter.hpp

```diff
@@ -164,6 +166,13 @@ class ReporterT final {
                         out << ",";
                         out << getter(event.second);
                         out << std::endl;
+
+                        // Log progress every 100e6 iterations
+                        if (++iter % (100 * 1000 * 1000) == 0) {
+                            BOOST_LOG_TRIVIAL(info)
```

Comment:
My only real comment is why did you chose to make the other messages debug level and this one info? I'd say that phase level should be debug and overall level should be info.

Reply:
We want to see this output on evergreen but we don't want debug logging on evergreen so info it is 😄